packages/puppeteer-extra-plugin-stealth/evasions/_utils/index.js

```diff
@@ -306,7 +306,7 @@ utils.replaceWithProxy = (obj, propName, handler) => {
   const originalObj = obj[propName]
   const proxyObj = new Proxy(obj[propName], utils.stripProxyFromErrors(handler))
 
-  utils.replaceProperty(obj, propName, { value: proxyObj })
+  utils.replaceProperty(obj, propName, { configurable: true, value: proxyObj })
```

Comment:
I'm not sure that's needed 🤔 from my understanding of the spec: descriptors that are not defined in `defineProperty` will simply be kept with their original value, also things that had `configurable: false` before can't be modified by us - haven't tested/confirmed the original issue yet though

Reply:
I can add the `descriptorOverrides` parameter to let the code calling `replaceWithProxy()` choose.

```JavaScript
utils.replaceWithProxy = (obj, propName, handler, descriptorOverrides = {}) => {
  const originalObj = obj[propName]
  const proxyObj = new Proxy(obj[propName], utils.stripProxyFromErrors(handler))

  utils.replaceProperty(obj, propName, { ...descriptorOverrides, value: proxyObj })
  utils.redirectToString(proxyObj, originalObj)

  return true
}
```